### PR TITLE
feat: optimize getMultipleAccounts

### PR DESCRIPTION
**Pull request by Dappio-emerson**
- Update `getMultipleAccounts` in `utils` by using web3.js internal function `_rpcBatchRequest()` to fetch multiple accounts at once.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -127,21 +127,79 @@ export const getTokenList = async () => {
...
-  for (let i = 0; i < publicKeys.length; i += BATCH_SIZE) {
-    let slices = publicKeys.slice(i, i + BATCH_SIZE);
-    let results = await connection.getMultipleAccountsInfo(slices);
-    let resultWithKeys = results.map((result, j) => {
-      return {
-        pubkey: slices[j],
-        account: result,
+  }[]
+> {
+  interface RPCResult {
```
Comment (ironaddicteddog) [inline src/utils.ts:142]: Move this to types

File: src/utils.ts [hunk 2]
```diff
@@ -127,21 +127,79 @@ export const getTokenList = async () => {
...
+
+  return results.map((res, index) => {
+    return {
+      pubkey: publicKeys[index],
+      account: res,
+    };
+  });
+}
+
+function chunks(array: PublicKey[], size: number) {
```
Comment (ironaddicteddog) [inline src/utils.ts:201]: Move this function inside of `getMultupleAccounts` as arrow function.

Ex:

```typescript=
const chunks = () => {
  // ...
}
```

File: src/index.ts [hunk 3]
```diff
@@ -8,6 +8,6 @@ export * as francium from "./francium";
 export * as orca from "./orca";
 export * as tulip from "./tulip";
 export * as nftFinance from "./nftFinance";
-export * as utils from "./utils";
```
Comment (ironaddicteddog) [inline src/index.ts:11]: Tbh this might be a better approach since it gives all util functions a shared domain. What's your take?

Comment (Dappio-emerson) [reply src/index.ts:11]: Agree, then I'll change it back and use `utils.getMultipleAccounts` in gateway.

**Pull request merged**